### PR TITLE
Added options for setting custom font and text colors.

### PR DESCRIPTION
**Pull request by sasojadrovski**
- Added properties for setting a custom font and text color for the
IQDropdownTextField items.
- Added property for setting a custom color for the optional item.
- Updated the Demo project to reflect the changes.

### CODE REVIEW

File: IQDropDownTextField Demo/Drop Down TextField/Drop Down TextField-Info.plist [hunk 1]
```diff
@@ -21,9 +21,13 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1.0.9</string>
+	<string>1.1.1</string>
```
Comment (hackiftekhar) [inline IQDropDownTextField Demo/Drop Down TextField/Drop Down TextField-Info.plist]: You should not increase build version of Info.plist file, could you please revert this change?

File: IQDropDownTextField/IQDropDownTextField.h [hunk 2]
```diff
@@ -143,6 +143,22 @@ extern NSInteger const IQOptionalTextFieldIndex;
  */
 @property(nullable, nonatomic,copy)   NSAttributedString     *attributedText NS_UNAVAILABLE;
 
+/**
+ Sets a custom font for the IQDropdownTextField items. Default is boldSystemFontOfSize:18.0.
+ */
+@property (nullable, strong, nonatomic) UIFont *font;
```
Comment (hackiftekhar) [inline IQDropDownTextField/IQDropDownTextField.h]: Looks like `font` & `textColor` property already exist with UITextField, but we can use `optionalItemTextColor`. However unintentionally you've encouraged me to use UITextField property to render texts in within UIPickerView. Thanks for this 👍

Comment (hackiftekhar) [inline IQDropDownTextField/IQDropDownTextField.h]: Hmm, now I think we should use different property name for dropDown texts and don't use the native once. So we may rename `font` to `dropDownFont`, `textColor` to `dropDownTextColor`.

Comment (sasojadrovski) [inline IQDropDownTextField/IQDropDownTextField.h]: Do you want me to just change the names of the properties?
